chat/UserView.vue

```diff
@@ -244,6 +244,9 @@
     @Prop({ default: false })
     readonly avatar: boolean = false;
 
+    @Prop({ default: false })
+    readonly avatarOrig: boolean = false;
```

Comment:
For an outward facing parameter, I don't think this is a good name. Something like ``useOriginalAvatar`` gets the point across easier, especially since our current setup won't allow for TSDoc-style documentation to properly work in Vue SFCs.